### PR TITLE
DUNT - Antenna and receiver swap

### PR DESCRIPTION
**Pull request by aleks-spes**
Antenna and receiver swaps as a part of GNSS@TG project.

### CODE REVIEW

**Review by EvanCGNS — CHANGES_REQUESTED** [review]
Looks good, just need to update `install/firmware.csv` as it is a new receiver

File: install/sessions.csv [hunk 1]
```diff
@@ -107,7 +107,8 @@ DUND,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-02-26T11:
 DUND,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2015-03-08T22:32:30Z,2019-10-01T00:07:13Z
 DUND,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,linz,trimble_netr9 x4,2019-10-01T00:07:14Z,9999-01-01T00:00:00Z
 DUNT,P Denys,OUSD,LEICA CRS1000,GPS,30s,4,sagenz,leica_lb2 x6,1999-09-23T00:00:03Z,2007-06-06T00:00:00Z
-DUNT,P Denys,OUSD,LEICA GRX1200GGPRO,GPS+GLO,30s,4,sagenz,leica_lb2 x6,2007-06-06T00:00:01Z,9999-01-01T00:00:00Z
+DUNT,P Denys,OUSD,LEICA GRX1200GGPRO,GPS+GLO,30s,4,sagenz,leica_lb2 x6,2007-06-06T00:00:01Z,2024-07-31T03:26:45Z
+DUNT,P Denys,OUSD,TRIMBLE ALLOY,GPS+GLO+GAL+BDS+QZSS,30s,0,sagenz,trimble_netr9 x4,2024-07-31T03:26:46Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/sessions.csv:111]: I think we should change the operator entry for 2024-07 onward (please change to "GeoNet,GNS")

**Review by elidana — COMMENTED** [review]
as discussed, please also update this and remove DUNT from list of third party sites
https://github.com/GeoNet/delta/blob/main/tools/gloria/main.go#L17

File: install/sessions.csv (hunk 1 above)
Comment (aleks-spes) [reply install/sessions.csv:111]: I thought we can update it after the Delta meeting for all "new" sites.

Comment (elidana) [reply install/sessions.csv:111]: no , that one would be related to the network code to be used for those sites, which is broader. This bit here is the one used to indicate who is the operator generating the RINEX data, so I think is appropriate to change it right away, as data will be now generated by GeoNet (regardless of the netwrok attribution of the site, if it makes sense)

Comment (aleks-spes) [reply install/sessions.csv:111]: @elidana what do you think about that part `sagenz`?

Comment (elidana) [reply install/sessions.csv]: ah good spot! missed that one. I think that field is used to select what COMMENT to use in the RINEX headers as well, though in the protobuf generation we probably did not included `sagenz` as we were not collecting any of those in gloria.

I suggest to use `geonet` for now, but remember to eventually change it to `linz` once we revisit the discussion about network code attribution within delta.

Alternatively, just put linz right on. We have similar cases for those in delta, see for example CNCL
```
CNCL,P Denys,OUSD,TRIMBLE 4000SSI,GPS,30s,4,sagenz,trimble_r x21,2000-02-05T00:00:00Z,2007-02-01T20:00:00Z
CNCL,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,sagenz,trimble_5700 x4,2007-02-01T20:00:01Z,2014-02-04T19:00:00Z
CNCL,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,3,geonet,trimble_netr9 x4,2014-02-04T19:00:01Z,2014-07-09T23:59:59Z
CNCL,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2014-07-10T00:00:00Z,2022-10-18T19:18:31Z
CNCL,GeoNet,GNS,TRIMBLE ALLOY,GPS+GLO,30s,0,geonet,trimble_netr9 x4,2022-10-18T19:18:32Z,9999-01-01T00:00:00Z
```

**Comment by aleks-spes** [comment]
Hey @junghao! We have to enable data downloading from a new station `DUNT` which used to be a third-party station. Could you please have a look at the Gloria-related part?

**Review by EvanCGNS — COMMENTED** [review]
Assuming the antenna serial number on the ticket is right, LGTM!

**Review by EvanCGNS — APPROVED** [review]

**Review by junghao — APPROVED** [review]
For Gloria part it's the right change.

**Comment by aleks-spes** [comment]
Hey @elidana ! Could you please review it?

**Review by elidana — APPROVED** [review]
looks good, thanks @aleks-spes !

**Pull request merged**